images/Youtube Video Page.html

```diff
@@ -0,0 +1,249 @@
+                            </div>
+                            <div class="comment">Lorem ipsum    dolor sit amet consectetur adipisicing elit. In alias id eos nobis nihil cum quae vero placeat officiis quia fuga et ratione, qui enim amet ipsa beatae earum laborum?</div>
+                            <div class="comment-icons">
+                                <img src="./Youtube Video Page_files/thumbs-up-icon.jpg" alt="thumbs up" height="30">
+                                <img src="./Youtube Video Page_files/thumbs-down-icon.png" alt="thumbs down" height="20">
+                                <div><b>Reply</b></div>
+                            </div>
+                            
+                        </div>
+                    </div>
+                    <div>
+                        <img src="./Youtube Video Page_files/user-image.png" alt="user image" height="60">
+                        <div>
+                            <div>
+                                <span class="commentersName"><b>Esther Andrews</b></span>
+                                <span class="time-of-comment">2 hours ago</span>
+                            </div>
+                            <div class="comment">Lorem ipsum    dolor sit amet consectetur adipisicing elit. In alias id eos nobis nihil cum quae vero placeat officiis quia fuga et ratione, qui enim amet ipsa beatae earum laborum?</div>
+                            <div class="comment-icons">
+                                <img src="./Youtube Video Page_files/thumbs-up-icon.jpg" alt="thumbs up" height="30">
+                                <img src="./Youtube Video Page_files/thumbs-down-icon.png" alt="thumbs down" height="20">
+                                <div><b>Reply</b></div>
+                            </div>
+                            
+                        </div>
+                    </div>
+                    <div>
+                        <img src="./Youtube Video Page_files/user-image.png" alt="user image" height="60">
+                        <div>
+                            <div>
+                                <span class="commentersName"><b>Esther Andrews</b></span>
+                                <span class="time-of-comment">2 hours ago</span>
+                            </div>
+                            <div class="comment">Lorem ipsum    dolor sit amet consectetur adipisicing elit. In alias id eos nobis nihil cum quae vero placeat officiis quia fuga et ratione, qui enim amet ipsa beatae earum laborum?</div>
+                            <div class="comment-icons">
+                                <img src="./Youtube Video Page_files/thumbs-up-icon.jpg" alt="thumbs up" height="30">
+                                <img src="./Youtube Video Page_files/thumbs-down-icon.png" alt="thumbs down" height="20">
+                                <div><b>Reply</b></div>
+                            </div>
+                            
```

Comment:
The section tag would be better for your comments, not the div tag. Section is more semantically correct and makes your code more readable.